apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -226,7 +226,7 @@ Keep in mind you are not just generating an example; you are generating an actua
     job.data['completion'] = completion;
 
     if (html && typeof html === 'string') {
-      const errors = validateHTML(html, false);
+      const errors = validateHTML(html, false, typeof job?.data?.python === 'string');
```

Comment:
This will return truthy for the empty string. Consider `!!job?.data?.python` instead, which will evaluate to `true` only if it's a non-empty string.

Reply:
done in 05c1b45.